[2022-07-08 13:38:00,972] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: my_dag.accurate manual__2022-07-08T13:37:58.003866+00:00 [queued]>
[2022-07-08 13:38:00,978] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: my_dag.accurate manual__2022-07-08T13:37:58.003866+00:00 [queued]>
[2022-07-08 13:38:00,978] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-07-08 13:38:00,978] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-07-08 13:38:00,978] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-07-08 13:38:00,984] {taskinstance.py:1270} INFO - Executing <Task(BashOperator): accurate> on 2022-07-08 13:37:58.003866+00:00
[2022-07-08 13:38:00,988] {standard_task_runner.py:52} INFO - Started process 662 to run task
[2022-07-08 13:38:00,990] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'accurate', 'manual__2022-07-08T13:37:58.003866+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/model.py', '--cfg-path', '/tmp/tmpjci2thfb', '--error-file', '/tmp/tmpd6uivc64']
[2022-07-08 13:38:00,991] {standard_task_runner.py:80} INFO - Job 13: Subtask accurate
[2022-07-08 13:38:01,024] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.accurate manual__2022-07-08T13:37:58.003866+00:00 [running]> on host 18ea9465fe18
[2022-07-08 13:38:01,058] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=accurate
AIRFLOW_CTX_EXECUTION_DATE=2022-07-08T13:37:58.003866+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-08T13:37:58.003866+00:00
[2022-07-08 13:38:01,059] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-07-08 13:38:01,059] {subprocess.py:74} INFO - Running command: ['bash', '-c', "echo 'accurate' "]
[2022-07-08 13:38:01,068] {subprocess.py:85} INFO - Output:
[2022-07-08 13:38:01,069] {subprocess.py:89} INFO - accurate
[2022-07-08 13:38:01,069] {subprocess.py:93} INFO - Command exited with return code 0
[2022-07-08 13:38:01,085] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=accurate, execution_date=20220708T133758, start_date=20220708T133800, end_date=20220708T133801
[2022-07-08 13:38:01,122] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-07-08 13:38:01,141] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
